pull-requests.md

```diff
@@ -50,6 +50,8 @@ If these things are not true then you should address the blockers before raising
 ## Descriptive
 A PR's description should include sufficient information on what the change is and why it is necessary.
 
+You should self-review your PR and leave comments to pre-empt any questions that are likely to come up at the right moment, or draw attention to areas that need more attention.
+
```

Comment:
I like having this suggestion, but I wonder would it be better placed down a couple of paragraphs? Here it interrupts the flow of what was there, meaning that the ”future peace of mind” bit now refers to the self-review, where it used to refer to having a PR at all. I would be tempted to place it as the last paragraph under this heading.

Reply:
(sorry I didn’t get to this before merge!)